Content.Client/Shuttles/UI/ShuttleNavControl.xaml.cs

```diff
@@ -303,7 +303,7 @@ private void DrawDocks(DrawingHandleScreen handle, EntityUid uid, Matrix3x2 grid
         const float sqrt2 = 1.41421356f;
         const float dockRadius = DockScale * sqrt2;
         // Worst-case bounds used to cull a dock:
-        Box2 viewBounds = new Box2(-dockRadius, -dockRadius, Size.X + dockRadius, Size.Y + dockRadius);
+        Box2 viewBounds = new Box2(-dockRadius * UIScale, -dockRadius * UIScale, (Size.X + dockRadius) * UIScale, (PixelSize.Y + dockRadius) * UIScale);
```

Comment:
```suggestion
        Box2 viewBounds = new Box2(-dockRadius * UIScale, -dockRadius * UIScale, (Size.X + dockRadius) * UIScale, (Size.Y + dockRadius) * UIScale);
```
you are calculating X and Y coordinates differently
pixelsize already includes the UI scale, so I assume you meant this one?
![grafik](https://github.com/user-attachments/assets/8d0844e0-b773-4b9d-8bc5-ae13ecf83ac6)

Reply:
I can't tell what's rider's AI shit gaslighting me or typos anymore.